frontend/src/shared/configuration/shared.config.js

```diff
@@ -1,6 +1,7 @@
 export const SHARED_CONFIG = {
   api: {
-    url: '#{SERVER_API_URL}#',
+    // url: '#{SERVER_API_URL}#',
+    url: 'https://wheelingfulapi.azurewebsites.net/',
```

Comment:
Please, don't commit your local changes for any config values.

Reply:
Fixed